### ISSUE TITLE
cant play

### ISSUE THREAD

**Issue by Noext** [mistertest/xcloud-shield#32]
![PXL_20220506_105643008 MP](https://user-images.githubusercontent.com/1674541/167120300-1d0985e1-953e-4df9-9eee-34282cd9f584.jpg)

got this errror every time, official app is working
running on shield tv pro last gen brand new

**Comment by tjukkie** [comment]
Yes, same problem, games won't start, unable to play like this. Any solution?

**Comment by mistertest** [comment]
Hi excuse me for time, I was busy with my other project CloudChoice. So I will woking on this issues and for building a new version this week. I will need help for test and feellback soon. Thanks guys.

**Comment by tjukkie** [comment]
No problem, will check back here and test new version when available.

**Comment by mistertest** [comment]
So I have good news. I tested the app in my shield tv, and I not played on it with xcloud or gfn since long times. So first surprise fpr me was the new interface. After that for every games I launched I has the same issue than your. I think it's about the new version included more tactils and the signal of the controler when the game start.  So I found a trick for bypassed this issue for every games you launched. When you launch a game, just when the big logo xbox is appear, make a long press "button A " till you see the screen of starting of the game. I tested the trick for a lot of games and it's working very well, I have no issues with this trick. So just guys you'll have to do the same for every games you launch. Confirm me if it's working for you I think it will be the case 99%

**Comment by tjukkie** [comment]
> So I have good news. I tested the app in my shield tv, and I not played on it with xcloud or gfn since long times. So first surprise for me was the new interface. After that for every games I launched I has the same issue than your. I think it's about the new version included more tactils and the signal of the controler when the game start. So I found a trick for bypassed this issue for every games you want to launch. When you launch a game, just when the big logo xbox is appear, make a long press "button A " till you see the screen of starting of the game. I tested the trick for a lot of games and it's working very well, I have no issues with this trick. So just guys you'll have to do the same for every games you launch. Confirm me if it's working for you I think it will be the case 99%

That would be nice. Going to test later tonight. Thanks for the info.

**Comment by Noext** [comment]
its working
but quality is very poor, running on a 4k tv with 1gbps fiber, steam look like 720p

**Comment by mistertest** [comment]
But I already know for the 4K since long time. But i'm focalised to the issue playing and the trick with "Button A " pressed is working. I wait others confirmations.

**Comment by tjukkie** [comment]
Tried and works :) Not a 4K setup here :) Sound and latency are good. Thanks for this tip. @mistertest
